dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-4.1/resources/l10n/com.ibm.websphere.appserver.microProfile-4.1_de.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=Dieses Feature ist eine Kombination der Liberty-Features, die Eclipse MicroProfile 4.1 f\u00fcr Cloud Native Java unterst\u00fctzen.
```

Comment:
```suggestion
description=Dieses Feature ist eine Kombination der Liberty-Features, die MicroProfile 4.1 f\u00fcr Cloud Native Java unterst\u00fctzen.
```

Reply:
Please replace Eclipse MicroProfile with MicroProfile as after MicroProfile Working group was set up, the name no longer needs this prefix of Eclipse.